doc/conf.py

```diff
@@ -248,3 +249,6 @@
     "github_version": "main",
     "commit": commit_link,
 }
+
+# Configurations for LaTeX
+latex_engine = "xelatex"
```

Comment:
The default latex engine is `pdflatex`, and it doens't support Unicode characters well. With `latex_engine` set to `xelatex`, Sphinx adds more tex packages and supports Unicode better.